com/paypal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package com.paypal;
+
+public class Perfection {
+
+    private static Perfection perf;
```

Comment:
Give meaniful variable name, instead of "perf", you can refactor to "perfection"